### ISSUE TITLE
mobile friendly pages 

### ISSUE THREAD

**Issue by subtosharki** [SnailyCAD/snaily-cadv4#435]
### Describe the feature

mobile friendly pages , obviously some features may not be possible like livemap and many other . this project would take a while

**Comment by Dev-CasperTheGhost** [comment]
Most of the CAD is mobile friendly, the only area that needs work is the admin dashboard

**Comment by jxckUK** [comment]
I've noticed that the CAD navigation doesn't scale well / doesn't show at all on mobile devices to be honest. I use an iPhone 11 and it won't work with the CAD.

**Comment by Dev-CasperTheGhost** [comment]
Yeah, that's currently the correct behavior. I don't have a mobile nav setup yet.

**Comment by Dev-CasperTheGhost** [comment]
> I've noticed that the CAD navigation doesn't scale well / doesn't show at all on mobile devices to be honest. I use an iPhone 11 and it won't work with the CAD.

The navigation should now be quite a bit more responsive. See video at #442
